bindings/node.js/binding.gyp

```diff
@@ -0,0 +1,41 @@
+{
+  "targets": [
+    {
+      "target_name": "kzg",
```

Comment:
It's fine to only support osx for now. But this could be behind a platform-specific conditions block.

Reply:
A lot more work to be done to support multiple platforms. Eventually maybe something more like https://github.com/ChainSafe/blst-ts

I will get it working on OSX for my own devexp and one flavor of linux for our testnet...